cds/modules/migrator/records.py

```diff
@@ -778,9 +777,6 @@ def _create_missing_subformats(cls, record, deposit):
         max_width = int(master['tags']['width'])
         max_height = int(master['tags']['height'])
 
-        if ratio not in current_app.config['CDS_SORENSON_PRESETS']:
-            ratio = get_closest_aspect_ratio(max_height, max_width)
-
         assert ratio in current_app.config['CDS_SORENSON_PRESETS']
```

Comment:
I think this one will fail if the ratio is not in the sorenson presets, right?

Reply:
oh didnt see that. Yes of course so should I remove it?